source/module_io/input_conv.cpp

```diff
@@ -767,6 +767,35 @@ void Input_Conv::Convert(void)
     GlobalV::qo_strategy = INPUT.qo_strategy;
     GlobalV::qo_thr = INPUT.qo_thr;
     GlobalV::qo_screening_coeff = INPUT.qo_screening_coeff;
+
+    //-----------------------------------------------
+    // PEXSI related parameters
+    //-----------------------------------------------
+    GlobalV::pexsi_npole = INPUT.pexsi_npole;
```

Comment:
define the global variables as static members of pexsi class, we will try to minimize global variables in future, so do not add new ones

Reply:
moved to `hsolver::DiagoPexsi`